docs/API.md

```
@@ -0,0 +1,122 @@
+  "successRate24h": 0,
+  "successRate7d": 0.9,
+  "avgConfidenceScore": 0,
+  "improvementTrend": 0.2,
+  "lastCalculated": 1700000000000
+}
+```
+
+### POST /classify
+Classify a gesture from raw landmarks. Falls back to cloud recognition when the local model is unavailable.
+
+**Body**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)